<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_registration</class>
 <widget class="QDialog" name="window_registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>259</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>381</width>
    <height>2</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>32423</width>
    <height>234234</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>381</width>
     <height>251</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>381</width>
     <height>251</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 5px;
background-color: #2a2a2a;
color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>366</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;</string>
    </property>
    <property name="text">
     <string>–ó–∞–ø–æ–ª–Ω–∏—Ç–µ –ø–æ–ª—è —Ä–µ–≥–∏—Å—Ç—Ä–∞—Ü–∏–∏</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>175</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 14px;
	background-color: #404040;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
    </property>
    <property name="text">
     <string>–ó–∞—Ä–µ—Å—Ç—Ä–∏—Ä–æ–≤–∞—Ç—å—Å—è</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>226</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #404040;
border-radius: 5px;
padding: 2px 5px;
font-size: 14px;</string>
    </property>
    <property name="placeholderText">
     <string>–õ–æ–≥–∏–Ω</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>55</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 22px;</string>
    </property>
    <property name="text">
     <string>–†–µ–≥–∏—Å—Ç—Ä–∞—Ü–∏—è</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>226</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #404040;
border-radius: 5px;
padding: 2px 5px;
font-size: 14px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>–ü–∞—Ä–æ–ª—å</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>36</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	padding-top: 5px;
	font-size: 22px;
	background-color: #404040;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
    </property>
    <property name="text">
     <string>ü°∏</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	background-color: #404040;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>5</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 20px;
	background-color: #f73643;
	border-radius: 8px;
}
QPushButton::hover {
	background-color: #d63a44;
	color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="hide">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>5</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-bottom: 4px;
	font-size: 20px;
	background-color: #47fc65;
	border-radius: 8px;
}
QPushButton::hover {
	background-color: #49cc5f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>5</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
color: white;</string>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
